core/src/Kokkos_ExecPolicy.hpp

```diff
@@ -864,6 +864,178 @@ ThreadVectorRange(const TeamMemberType&, const iType1& arg_begin,
 
 namespace Impl {
 
+enum class TeamMDRangeLastNestLevel : bool { NotLastNestLevel, LastNestLevel };
+enum class TeamMDRangeParThread : bool { NotParThread, ParThread };
+enum class TeamMDRangeParVector : bool { NotParVector, ParVector };
+enum class TeamMDRangeThreadAndVector : bool { NotBoth, Both };
+
+struct NoReductionTag {
+  explicit NoReductionTag() = default;
```

Comment:
What's the point in adding a default constructor (and why make it `expiicit`)?

Reply:
Changed in [f5f24e3](https://github.com/kokkos/kokkos/pull/5238/commits/f5f24e33bb5c604fb0cbc471850d201f86ca308c).

Removed this explicit constructor.